homework5.py

```diff
@@ -0,0 +1,30 @@
+# 1. Створити програму, яка буде очікувати від користувача введення тексту і виведе інформацію по кожному надрукованому символу:
+# це “число” + яке воно (парне, непарне),
+# це “буква” + яка вона (велика чи маленька),
+# це “символ”
+
+user_input = input('Type something: ')
+for item in user_input:
+    if item.isdigit():
+        if int(item) % 2 == 0:
+            print(f'This is digit {item}. It is even.')
+        else:
+            print(f'This is digit {item}. It is odd.')
+    elif item.isalpha():
+        if str(item).istitle():
```

Comment:
There is no need to transform `item` to `str()` here since it is already a string. Also, you can check if it is uppercase using `isupper()`. The difference is that `istitle()` checks if the first character in a string is Uppercase while other are lowercase (if len of the string is greater that 1), when `isupper()` checks if all characters of a sting are uppercase

Reply:
Thanks for detailed explanation.  I improved the code.